dcpy/utils/geospatial.py

```diff
@@ -22,6 +40,72 @@
 }
 
 
+def convert_to_geodata(
+    data: pd.DataFrame,
+    geometry_format: str,
+    geometry_column: str,
+    new_geometry_column: str = "geometry",
+) -> gpd.GeoDataFrame:
+    data = data.copy()
+    match geometry_format:
+        case "WKT":
+            data[geometry_column] = gpd.GeoSeries.from_wkt(data[geometry_column])
+        case "WKB":
+
+            def _try_wkb(wkb):
```

Comment:
Just curious, why not to use `GeoSeries.from_wkb`?

Reply:
sorry I shouldn't put a comment in this section explaining why it's so weird

see [4050fc7](https://github.com/NYCPlanning/data-engineering/pull/367/commits/4050fc72f87d76a31b85e30e8d0f50765826702d). the same may be true for `from_wkt` as well so I'll test that